docroot/sites/all/modules/features/build/midcamp_fields/midcamp_fields.features.field_base.inc

```diff
@@ -1445,6 +1575,31 @@ Prof.',
     'type' => 'list_boolean',
   );
 
+  // Exported field_base: 'field_ticket_id'.
+  $field_bases['field_ticket_id'] = array(
```

Comment:
This is ticket_class_id. Should this actyually be a list with a callback function that maps the type? Or is this their purchased ticket's ID?